library/src/main/java/com/schibsted/spain/barista/custom/DisplayedMatchers.java

```diff
@@ -0,0 +1,31 @@
+package com.schibsted.spain.barista.custom;
+
+import android.support.annotation.IdRes;
+import android.support.annotation.NonNull;
+import android.support.annotation.StringRes;
+import android.view.View;
+
+import org.hamcrest.Matcher;
+
+import static android.support.test.espresso.matcher.ViewMatchers.isDisplayed;
+import static android.support.test.espresso.matcher.ViewMatchers.withId;
+import static android.support.test.espresso.matcher.ViewMatchers.withText;
+import static org.hamcrest.core.AllOf.allOf;
+
+public class DisplayedMatchers {
```

Comment:
Let me change the order of the matchers, to make them more readable if possible :·D